packages/components/pie-button/custom-elements.json

```diff
@@ -0,0 +1,509 @@
+{
+  "schemaVersion": "1.0.0",
```

Comment:
do we still want to gitignore these files? 😄

Reply:
Also happy to add the docs. I wonder if it's worth moving over to GH Wiki though for better visibility? cc @ashleynolan

Reply:
ahh I see, thank you!

Reply:
I think this makes sense in the wiki too, as that'd be clearer for people using the repo